cloud/db_cloud_impl.cc

```diff
@@ -143,6 +143,12 @@ Status DBCloud::Open(const Options& opt, const std::string& local_dbname,
     if (read_only) {
       return Status::NotFound("CLOUDMANIFEST not found and read_only is set.");
     }
+    if (cenv->GetCloudEnvOptions().cookie_on_open !=
+        cenv->GetCloudEnvOptions().new_cookie_on_open) {
+      Log(InfoLogLevel::WARN_LEVEL, options.info_log,
+          "Switch CLOUDMANIFEST is not supported when opening new db. Only "
+          "cookie_on_open will be used");
```

Comment:
Why? You should always write with `new_cookie_on_open`, whether the DB exists or not.

Reply:
`cookie_on_open` means the cookie we will open db with. `new_cookie_on_open` is the cookie that we switch to. For a brand new db, there is no need to switch cookie since even `CLOUDMANIFEST-cookie_on_open` doens't exist

Reply:
I look at it as:

- you open the existing CLOUDMANIFEST with `cookie_on_open`, if it doesn't exist => new DB
- you write new CLOUDMANIFESTs with `new_cookie_on_open`

so you always read one and write the other

Reply:
"New DB" is not a special case. Every time you open, you read an existing DB (if it exists) and write under a new name.

Reply:
I see. Yeah. That sounds more intuitive.